crates/matrix-sdk/src/room/timeline/inner.rs

```diff
@@ -368,12 +370,93 @@ impl<P: ProfileProvider> TimelineInner<P> {
             .await;
         }
     }
+
+    fn update_event_item(&self, index: usize, event_item: EventTimelineItem) {
+        self.items.lock_mut().set_cloned(index, Arc::new(TimelineItem::Event(event_item)))
+    }
 }
 
 impl TimelineInner {
     pub(super) fn room(&self) -> &room::Common {
         &self.profile_provider
     }
+
+    pub(super) async fn fetch_in_reply_to_details(
```

Comment:
I don't see this function being called anywhere. Am I looking at a wrong diff?

Reply:
It is called in `fetch_event_details`.

Reply:
Ah, I marked the file that contains it as reviewed so it was collapsed 😅